tests/_utils/stop_tidb_cluster

```diff
@@ -6,3 +6,24 @@ killall -w -s 9 tidb-server || true
 killall -w -s 9 cdc || true
 killall -w -s 9 cdc.test || true
 killall -w -s 9 tiflash || true
+
+kill -9 $(lsof -i tcp:4000 -t) &>/dev/null || true
```

Comment:
these ports may be customed by env value, https://github.com/pingcap/ticdc/blob/master/tests/_utils/test_prepare.
Could we use these variables instead of the port number directly?

Reply:
fine